### PR TITLE
contrib: Clean up un-used scripts

### PR DESCRIPTION
**Pull request by sayboras**

### CODE REVIEW

**Review by qmonnet — APPROVED** [review]
Sounds good to me, I'm not aware of any of these scripts being in use today. But I asked for a review from maintainers, in case they'd be aware of remaining use cases.

File: contrib/shell/test.sh
Comment (joestringer) [inline contrib/shell/test.sh:31]: I happen to find this helper super useful, but I already had it in my `.bashrc` and I was the one who added this into the tree to share the example with others :) I don't mind removing it if folks are discovering these tricks through other mechanisms.

One terminal: Run this command to retrigger tests whenever a file changes, then open your editor and make changes. Terminal reruns the test every time you save the file. You can iterate quite quickly on unit tests with this pattern.

**Review by joestringer — APPROVED** [review]
Most of these look like old relics from earlier bash-based CI or tricks to help deploy software that wasn't widely available years ago.

`watchtest` / `watchdo` are still part of my regular rotation, but I've integrated those into my own environment so it doesn't make a difference to me whether we continue to share those scripts via the Cilium repo or not.

File: contrib/shell/test.sh
Comment (qmonnet) [reply contrib/shell/test.sh:31]: [I had something similar based on inotifywait, too.]

I guess it's not a bad idea to keep these helpers somewhere for the reason you describe, but I wonder how we might make them more discoverable? Would it make sense to document them somewhere in the docs for contributing?

Comment (joestringer) [reply contrib/shell/test.sh:31]: We could. I think it's fair to say that this is out-of-scope of the actual goals of the project, but given the usefulness it's nice to have this example hosted somewhere. We could equally use a gist or such for this. So far I think the bar for `contrib/` directory has just been "does any developer find this useful?", and just put things there. From time to time we may share about tips/tricks naturally on Slack or during community meetings. If you think it's worth documenting in the contribution docs, we could also do that.

Comment (sayboras) [reply contrib/shell/test.sh:31]: seems like there is no strong objection to remove these files, I will merge this in and keep an eye if there is any users complaining about the removal.

**Pull request merged**
